newsrevenuehub/rev-engine
# [NRH-253] Links in richtext

#### What's this PR do?
Makes some updates the RichText editor including support for:
1. Links (the primary goal here)
2. Text-alignment
3. A few other inline text things

#### How should this be manually tested?
Edit an existing RichText block or create a new one on a donation page.
Notice the new features. Test links in particular.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
